scripts/3rdparty.py

```diff
@@ -554,6 +554,28 @@ def makeenv(envlist):
     execute('make -j', logfile)
     install_files('src/sox.h', os.path.join(builddir, 'include'))
     install_files('src/.libs/libsox.a', os.path.join(builddir, 'lib'))
+
+elif name == 'libunwind':
+	download(
+		'http://download.savannah.nongnu.org/releases/libunwind/libunwind-%s.%s.%s.tar.gz' %(ver, ver, ver),
```

Comment:
This code produces an incorrect url and paths, e.g.:

```
http://download.savannah.nongnu.org/releases/libunwind/libunwind-1.2.1.1.2.1.1.2.1.tar.gz
```

It also has other problems:

* on my system libunwind fails to build with -fvisibility=hidden
* on alpine libunwind fails to build without --disable-coredump
* the installation step does not work for me (make install prefix=src)

Here is a fixed version that works for me:

```
elif name == 'libunwind':
    download(
        'http://download.savannah.nongnu.org/releases/libunwind/libunwind-%s.tar.gz' % ver,
        'libunwind-%s.tar.gz' % ver,
        logfile,
        vendordir)
    extract('libunwind-%s.tar.gz' % ver,
            'libunwind-%s' % ver)
    os.chdir('src/libunwind-%s' % ver)
    execute('./configure --host=%s %s %s %s' % (
        toolchain,
        makeenv(envlist),
        makeflags(workdir, toolchain, deplist, variant=variant),
        ' '.join([
            '--enable-static',
            '--disable-shared',
            '--disable-coredump',
            '--disable-ptrace',
            '--disable-setjmp',
           ])), logfile)
    execute('make -j', logfile)
    install_files('include/*.h', os.path.join(builddir, 'include'))
    install_files('src/.libs/libunwind.a', os.path.join(builddir, 'lib'))
```

BTW, please use spaces instead of tabs and remove spurious empty lines.